Preprocessing in Spark.md

```diff
@@ -0,0 +1,175 @@
+# Preprocessing in Spark
+
+One of the major limitations of Pandas is that Pandas was designed for small datasets that can be handled on a single machine and thus it does not scale well to big data. On the contrary, Apache spark was designed for big data, but it has a different API and also lacks many of the easy-to-use functionality in Pandas for data wrangling and visualization.
+
+
+## 1. Loading data
+After setup a SparkSession we can read out dataset using set of functions read.option.csv:
+
+    df_spark=data_spark.read.option('header','true').csv('airlines.csv')
+
+We might see the top 20 rows of dataset using the show() method:
+
+    df_spark.show
+
+
+## 2. Exploring data
+
+### 2.1. Schema of data
+Typically, the first step to explore a DataFrame is to understand its schema: column names and corresponding data types.
+Using printSchema() function we might check what type of data type and columns of our dataset holds and if these columns consisting any null values:

```

Comment:
```suggestion
Using `printSchema()` function we might check what type of data type and columns of our dataset holds and if these columns consisting any null values:

```

Reply:
Poprawione

Reply:
ale tylko w tym miejscu. Dobrze by było przyjąć za zasadę, że wszystkie kody w tym nazwy funkcji są opakowane w blok kodu. Te w tekście w ten pojedynczy. 
Dla przykładu - gdzieś dalej jest :
* Pyspark’s **describe**() 
* similar function to fillna() from Pandas

Reply:
Ok, zrozumiałam. Poprawione